sanity/squashfs.go

```diff
@@ -108,10 +108,7 @@ func checkSquashfsMount() error {
 	}
 
 	// the fstype can be squashfs or fuse.{snap,squash}fuse
-	fstype, _, err := squashfs.FsType()
-	if err != nil {
-		return err
-	}
+	fstype, _ := squashfs.FsType()
```

Comment:
I think you could get the options as well and pass them to the logic below.

Reply:
Probably yes, although this would affect the options with "ro", "x-gdu.hide" and affect other tests, therefore I'd hold off and leave it as a followup material.